src/noit_lmdb_tools.c

```diff
@@ -284,6 +284,21 @@ noit_lmdb_instance_t *noit_lmdb_tools_open_instance(char *path)
     return NULL;
   }
 
+  /* Align mapsize to pagesize, use enough pages to reach
+   * 64 MB */
+  size_t page_size = getpagesize();
+  size_t map_size = page_size;
+  static const size_t map_size_target = (1024*1024*64);
+  while (map_size < map_size_target) {
+    map_size += page_size;
+  }
+  rc = mdb_env_set_mapsize(env, map_size);
```

Comment:
can just set this to 64 MiB here directly

Reply:
If we do that, how do we know that aligns with the local pagesize?

Reply:
map_size = page_size * ((map_size_target + (page_size-1)) / page_size);

Reply:
You don't need to do that yourself, lmdb will align the size if it doesn't match the page size.

However if it wasn't, we know the page size would be a power of 2

Reply:
I used Patrick's suggestion, since these two comments conflict. Can change to something else if needed.